avocado/core/test.py

```diff
@@ -601,10 +601,11 @@ def _start_logging(self):
         self.log.addHandler(self._file_handler)
         self.log.propagate = False
 
-        # add the test log handler to the root logger so that
-        # everything logged while the test is running, for every
-        # logger, also makes its way into the test log file
-        logging.root.addHandler(self._file_handler)
+        # Adding the the test log FileHandler to the Avocado's root logger so
+        # that everything logged while the test is running, for every logger,
+        # also makes its way into the test log file
+        root = logging.getLogger('avocado')
```

Comment:
Nitpick: it would be much clearer if the new code did not use the `root` variable for the logger containing the `avocado` namespace.  This is about removing the use of the root logger but then mentioning `root` dilutes that point.